data/ml

```diff
@@ -13,7 +12,6 @@ codepoints:
 - 3337  # ഉ (U+0D09) MALAYALAM LETTER U
 - 3338  # ഊ (U+0D0A) MALAYALAM LETTER UU
 - 3339  # ഋ (U+0D0B) MALAYALAM LETTER VOCALIC R
-- 3340  # ഌ (U+0D0C) MALAYALAM LETTER VOCALIC L
```

Comment:
I am waiting on confirmation from a native speaker/writer of Malayalam to confirm that this can go.